packages/serverpod/lib/src/server/server.dart

```diff
@@ -249,6 +249,13 @@ class Server {
       request.response.statusCode = result.statusCode;
       await request.response.close();
       return;
+    } else if (result is ExceptionResult) {
+      request.response.headers.contentType = ContentType.json;
+      request.response.statusCode = HttpStatus.internalServerError;
```

Comment:
Just an idea... I think it would be cool, if the user could set a different status code.

Reply:
It makes sense for this server to be used by multiple clients. However, if the exceptions are passed from the server to the client, it would be nonsensical.